python/tk_multi_launchapp/base_launcher.py

```diff
@@ -334,3 +339,35 @@ def launch_from_path_and_context(self, path, context, version=None):
         :param version: (Optional) Specific version of DCC to launch.
         """
         raise NotImplementedError
+
+
+    def _get_group_name(self, engine):
+        """
+        Construct a group name from an engine name. This name will be used to group
+        like commands together when registering commands with the current engine.
+
+        :param str engine: Toolkit engine name that the launch commands are associated with.
+        :returns: String group name or None
+        """
+        return engine.split("-")[-1].capitalize() if engine else None
+
+    def _sort_group_versions(self, versions):
+        """
+        Controls how version numbers are sorted. Implements the same methodology as Desktop.
```

Comment:
instead of saying "Implements the same methodology as Desktop.", it would be great to explain (maybe with an example?) of what it is doing?